File: cases/taint/cpp/argument-position-positive/argument_position_positive.cpp
int dfb_source() { // DFB-SOURCE: argument-position-input
    return 1;
}

int choose_first(int first, int second) { // DFB-WITNESS: argument-position-first
    return first;
}

void dfb_sink(int value) {} // DFB-SINK: argument-position-sink

void run() {
    int result = choose_first(dfb_source(), 0);
    dfb_sink(result);
}

File: cases/taint/php/argument-position-positive/argument_position_positive.php
<?php
function dfb_source(): string { // DFB-SOURCE: argument-position-input
    return "tainted";
}

function chooseFirst(string $first, string $second): string { // DFB-WITNESS: argument-position-first
    return $first;
}

function dfb_sink(string $value): void {} // DFB-SINK: argument-position-sink

function run(): void {
    $result = chooseFirst(dfb_source(), "clean");
    dfb_sink($result);
}
